### DISCUSSION TITLE
寻求一个快速启动方式服务器的方式

### DISCUSSION THREAD

**Discussion by FozuBlessNoBug** [kbengine/kbengine#1457] [Q&A]
本人新手菜鸟
做功能的时候，经常修改python代码，每次一个修改一点代码就需要重启整个服务器来验证，这样效率太低了，我去翻了一下有热更办法，试了一下有时候不生效。
有没有更好的办法？ 或者baseapp单独启动？而不重启整个服务器

**Comment by Wandar** [comment]
不生效是因为继承关系,父类需要先reload

**Comment by huhubis** [comment]
你网上搜一下python热更新

**Comment by FozuBlessNoBug** [comment]
> 不生效是因为继承关系,父类需要先reload

谢谢回复，目前有成熟的方案吗？

**Comment by FozuBlessNoBug** [comment]
> 你网上搜一下python热更新

谢谢回复，目前有成熟的方案吗？
